### PR TITLE
Add --region flag

### PR DESCRIPTION
**Pull request by m110**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# tdl
+
+This is the `tdl` CLI tool used for running the interactive trainings on the [Three Dots Labs Academy](https://academy.threedots.tech).
+
+## Install
+
+### Script (macOS, Linux)
+
+```sh
+sudo /bin/sh -c "$(curl -fsSL https://raw.githubusercontent.com/ThreeDotsLabs/cli/master/install.sh)" -- -b /usr/local/bin```
```
Comment (roblaszczak) [inline README.md:10]:
```suggestion
sudo /bin/sh -c "$(curl -fsSL https://raw.githubusercontent.com/ThreeDotsLabs/cli/master/install.sh)" -- -b /usr/local/bin
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# tdl
+
+This is the `tdl` CLI tool used for running the interactive trainings on the [Three Dots Labs Academy](https://academy.threedots.tech).
+
+## Install
+
+### Script (macOS, Linux)
```
Comment (roblaszczak) [inline README.md:7]:
```suggestion
### Script (macOS, Linux) - recommended
```

File: internal/update.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+const repoURL = "https://github.com/ThreeDotsLabs/cli"
+const releasesURL = "https://api.github.com/repos/ThreeDotsLabs/cli/releases/latest"
+
+func CheckForUpdate(currentVersion string) {
+	if currentVersion == "" || currentVersion == "dev" {
+		return
+	}
+
+	resp, err := http.Get(releasesURL)
```
Comment (roblaszczak) [inline internal/update.go:22]: IDK how long it takes, but maybe some timeout, like 100ms? 200ms? to not affect UX when it's slow for some reason

**Review by roblaszczak — APPROVED** [review]

File: internal/update.go (hunk 3 above)
Comment (m110) [reply internal/update.go:22]: 1a21eae4dbb4ba2d8d52bd2e69a3599d65598acf

**Pull request merged**